index.html

```diff
@@ -13,7 +13,7 @@
                 <div id="timer">
                     00:30
                 </div>                                        
-            </br>
+            </br> <!-- Comment -->
```

Comment:
All of these line breaks should be `<br>`.  They don't need the `/` you have in them and Firefox shows it as an error in the developer tools.

Reply:
I honestly didn't know this. I first learned HTML well over 10 years ago and I have this thing in my mind that it had to be formatted like that. Maybe it used to? Or i'm just mixing that up with a different tag. Thanks, I'll change them.

Reply:
If you were writing an XHTML document, you'd use `<br />` to denote that tag as not having a closing tag but no one writes XHTML.  In HTML, it's perfectly valid to use `<br>`.